cluster/config-defaults.yaml

```diff
@@ -289,11 +289,19 @@ enable_hpa_scale_to_zero: "true"
 
 # setup supporting components to enable encryption
 # this flag must only be switched from true to false when enable_encryption is false and all secrets were decrypted
+{{if eq .Environment "test"}}
+support_encryption: "true"
```

Comment:
I'd also rename the config item to something more specific (encryption of what?).

Reply:
I would defer this. The config item hopefully won't be here for long.